components/OrderSuccessContributorCardWithData.js

```diff
@@ -104,10 +105,11 @@ class OrderSuccessContributorCardWithData extends React.Component {
 
   render() {
     const { order, fromCollective, intl, data } = this.props;
-    const { totalAmount, interval, currency } = order;
+    const { interval, currency } = order;
     const { hasPopup } = this.state;
     const member = data && data.member;
     const { collectiveId, memberCollectiveId } = data && data.variables;
+    const totalAmount = order.data?.isFeesOnTop ? order.totalAmount - order.data.platformFee : order.totalAmount;
```

Comment:
I think `totalAmount` should be saved "ready to use" and not be modified.

Reply:
I disagree. The order total amount is correct, the order includes the fees in all cases.
I rather have an edge-case in the frontend than storing inaccurate data.
In the ideal scenario, we update the frontend to thank the user for the donation.

Reply:
Then maybe we should display the `totalAmount` on this card, without taking out the fees? If the final confirmation screen before paying says `Total: $10` it can be surprising to later see a different amount here

Reply:
The problem is that we display that inside a small collective card which totally suggests you donated the full amount to the collective.
I'll rethink this and start a discussion in Slack if needed so Design can be part of it.

Reply:
On the "success" page it doesn't bother me, the `Contributed a total of:` label is a good description and no one else is supposed to see this page.

![image](https://user-images.githubusercontent.com/1556356/85668126-a4042d00-b6be-11ea-9491-6270b9aa2aa6.png)

Reply:
Cool, I had in mind it was the collective card. I agree it's OK to display the total amount there, thanks!